core/escrow.js

```diff
@@ -37,58 +72,68 @@ const escrowExchange = {
         getRefund() {
           return refund.p;
         },
+        getUse() {
+          return E(escrowP).getUse();
```

Comment:
Will getUse() be defined by default? I thought the point of the new makeCustom code was to be able to add getUse() in just the cases where it was sensible, and not to have it in the default situation (i.e. fungible goods.)

Reply:
Good point. I'll try to take it out

Reply:
That's out!